op-guide/backup-restore.md

```diff
@@ -0,0 +1,116 @@
+---
+title: backup and restore
+category: advanced
+---
+
+# Backup and Restore
+
+## About
+
+This document describes how to back up and restore the data of TiDB. Currently, this document only covers full backup and restoration.
```

Comment:
```
s/back up/backup/
```

Although both are OK. But consistency should be considered in a single doc file.

Reply:
Agree. Please use backup.